src/fairchem/core/common/relaxation/optimizers/optimizable.py

```diff
@@ -0,0 +1,560 @@
+"""
+Copyright (c) Meta, Inc. and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+
+Code based on ase.optimize
+"""
+
+from __future__ import annotations
+
+from functools import cached_property
+from types import SimpleNamespace
+from typing import TYPE_CHECKING, ClassVar
+
+import numpy as np
+import torch
+from ase.calculators.calculator import PropertyNotImplementedError
+from ase.stress import voigt_6_to_full_3x3_stress
+from torch_scatter import scatter
+
+from fairchem.core.common.relaxation.ase_utils import batch_to_atoms
+
+# unreleased ASE has Optimizable, last released version 3.22.1 does not
+# thankfully we can get away with backwards compatibility by creating a dummy
+# submit an issue to the ASE repo to make new releases if you find this code ugly :(
```

Comment:
3.23 was released so this comment can be cleaned 
```suggestion

```

Reply:
thanks @CompRhys, updated the comment but will still leave the import exception in there until we pin the ASE version